db/schema.rb

```diff
@@ -70,6 +78,16 @@
     t.index ["content_id"], name: "index_link_sets_on_content_id", unique: true, using: :btree
   end
 
+  create_table "linkables", force: :cascade do |t|
+    t.integer  "content_item_id", null: false
+    t.string   "state",           null: false
+    t.string   "base_path",       null: false
+    t.string   "document_type",   null: false
+    t.datetime "created_at"
+    t.datetime "updated_at"
+    t.index ["content_id"], name: "index_link_sets_on_content_id", unique: true, using: :btree
+  end
```

Comment:
Ah, you're going through commit by commit. This has been removed in a later commit. I'll interactive-rebase them all into a single commit before merging.